### PR TITLE
fix(dashboard-dd): Add name of newly created dropdown

### LINKED ISSUE #15396: Create New Dashboard Field selection from Data Explorer is Blank
From Data Explorer, when save as a Dashboard Cell though "Save As" button.  When you select "Create a New Dashboard", the field goes blank so user is not sure if selected or not.  When completing fields, the dropdown field is still empty. 

newdashboard

### PR DESCRIPTION
**Pull request by ebb-tide**
Closes #15396

Kapture 2020-01-22 at 18 40 27

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/dataExplorer/components/DashboardsDropdown.tsx [hunk 1]
```diff
@@ -55,13 +56,18 @@ class DashboardsDropdown extends PureComponent<Props> {
...
+    const {dashboards, selectedIDs, newDashboardName} = this.props
 
     if (!selectedIDs.length) {
       return 'Choose at least 1 dashboard'
     }
 
-    return dashboards
+    const dashboardsWithNew = [
+      ...dashboards,
+      {...DashboardTemplate, name: newDashboardName},
```
Comment (drdelambre) [inline ui/src/dataExplorer/components/DashboardsDropdown.tsx]: that's pretty cool

File: ui/src/dataExplorer/components/DashboardsDropdown.tsx [hunk 2]
```diff
@@ -55,13 +56,18 @@ class DashboardsDropdown extends PureComponent<Props> {
...
+    ]
+
+    return dashboardsWithNew
```
Comment (glinton) [inline ui/src/dataExplorer/components/DashboardsDropdown.tsx]: could this have been:
```suggestion
    return [
      ...dashboards,
      {...DashboardTemplate, name: newDashboardName},
    ]
```
?

Looks good either way

Comment (ebb-tide) [reply ui/src/dataExplorer/components/DashboardsDropdown.tsx]: that would be better thanks!

Comment (ebb-tide) [reply ui/src/dataExplorer/components/DashboardsDropdown.tsx]: that is equivalent but I find that a bit harder to read.. ? thanks!

**Pull request merged**